common/src/JovoLogger.ts

```diff
@@ -1,120 +1,175 @@
-import {
-  Logger as TsLogger,
-  ISettingsParam,
-  TLogLevelName,
-  TUtilsInspectColors,
-  ILogObject,
-} from 'tslog';
-import _merge from 'lodash.merge';
-import _cloneDeep from 'lodash.clonedeep';
+import chalk, { Chalk } from 'chalk';
+import _mergeWith from 'lodash.mergewith';
+import { getLogger, levels, Logger, LogLevelDesc } from 'loglevel';
 import { JovoError } from './JovoError';
-import { inspect } from 'util';
-import { AnyObject } from './index';
-export class JovoLogger extends TsLogger {
-  constructor(settings?: ISettingsParam) {
-    super();
-    this.setSettings(_merge(this.settings, this.getDefaultConfig(), settings));
+
+export interface JovoLoggerConfig {
+  name: string | symbol;
+  level: LogLevelDesc;
+  disableStyling: boolean;
```

Comment:
I'm not 100% sure on the naming of that property.  `logging` includes a property called `styling`. `logger` uses `disableStyling`. I think we should be consistent here